### PR TITLE
resolved issue #2542 with position of map icon

### LINKED ISSUE #2598: Put map component in bottom sheet of upload activity
**Summary:** 
As discussed in #2542 Map component can be placed in the bottom sheet of upload activity  instead of the right side of the image

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by vanshikaarora**
**Removed MapIcon from right card and added it to the bottom card**

Fixes #2598 Put map component in bottom sheet of upload activity

### Changes made

- Modified upload_bottom_card.xml
- Removed toggle functions for right card layout
- Accordingly modified UploadActivity.java, UploadPresenter.java, UploadView.java

### CODE REVIEW

**Comment by vanshikaarora** [comment]
@maskaravivek @nicolas-raoul @ujjwalagrawal17 The PR is complete and up for review. Can you please review :)

File: app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java [hunk 1]
```diff
@@ -109,9 +109,8 @@
     @BindView(R.id.license_subtitle) TextView licenseSubtitle;
     @BindView(R.id.please_wait_text_view) TextView pleaseWaitTextView;
 
-    //Right Card
-    @BindView(R.id.right_card) CardView rightCard;
-    @BindView(R.id.right_card_expand_button) ImageView rightCardExpandButton;
+
+    //@BindView(R.id.right_card_expand_button) ImageView rightCardExpandButton;
```
Comment (domdomegg) [inline app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java]: Please can you remove the commented code - if we do need it later we can always go back into version control.

**Comment by maskaravivek** [comment]
@vanshikaarora Can you attach a screenshot?

**Comment by maskaravivek** [comment]
@vanshikaarora Can you resolve the merge conflicts?

**Comment by vanshikaarora** [comment]
Sure  @maskaravivek :) I just made the changes and the PR is ready for review :)

**Comment by vanshikaarora** [comment]
>@vanshikaarora Can you resolve the merge conflicts?

@maskaravivek I have resolved the merge conflicts Kindly review :)

**Comment by vanshikaarora** [comment]
>@vanshikaarora Can you attach a screenshot?

@maskaravivek here is a screenshot for the current position of map icon

WhatsApp Image 2019-03-17 at 3 39 45 PM

**Comment by vanshikaarora** [comment]
Thanks for approving the PR @maskaravivek  :)

**Pull request merged**